src/bootstrap/src/core/build_steps/gcc.rs

```diff
@@ -173,11 +190,17 @@ impl Step for Gcc {
 
         let lib_alias = install_dir.join("lib/libgccjit.so.0");
         if !lib_alias.exists() {
-            t!(builder.symlink_file(install_dir.join("lib/libgccjit.so"), lib_alias,));
+            t!(builder.symlink_file(&libgccjit_path, lib_alias));
         }
 
         t!(stamp.write());
 
-        true
+        GccOutput { libgccjit: libgccjit_path }
     }
 }
+
+/// Configures a Cargo invocation so that it can build the GCC codegen backend.
+pub fn add_cg_gcc_cargo_flags(cargo: &mut Cargo, gcc: &GccOutput) {
+    // Add the path to libgccjit.so to the linker search paths.
+    cargo.rustflag(&format!("-L{}", gcc.libgccjit.parent().unwrap().display()));
```

Comment:
We shouldn't prefer `display` to pass paths as strings IMO.

Reply:
How else to pass the Path then? :thinking:

Reply:
`to_str`?

Reply:
Ok, will change. I'm pretty sure that we use `display` on paths on a lot of places in bootstrap already though :sweat_smile: https://github.com/rust-lang/rust/blob/17472a9f80e6143538f5c29d9bbc0f1acf8dae4a/src/bootstrap/src/core/build_steps/compile.rs#L1298

Reply:
We should update them too (you don't have to do that in this PR).